src/pudl/transform/ferc1.py

```diff
@@ -4731,16 +4770,24 @@ class ElectricPlantDepreciationChangesFerc1TableTransformer(
     def process_xbrl_metadata(self, xbrl_metadata_json) -> pd.DataFrame:
         """Transform the metadata to reflect the transformed data.
 
-        Replace the name of the balance column reported in the XBRL Instant table with
-        starting_balance / ending_balance since we pull those two values into their own
-        separate labeled rows, each of which should get the original metadata for the
-        Instant column.
+        Warning: The calculations in this table are currently being checked using
+        reconcile_table_calculations(), but they still contain high rates of error.
+        This function replaces the name of the balance column reported in the XBRL
+        Instant table with starting_balance / ending_balance since we pull those two
+        values into their own separate labeled rows, each of which should get the
+        original metadata for the Instant column. We do this pre-processing before we
```

Comment:
This bit is a little hard to follow. Might be that I don't understand the context of the table, though.